_posts/2022-09-14-download-tool.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+author: Jennifer Chen
+title: New Data Download Tool
+---
+
+In the last year, we have added several interesting datasets and exciting new features to Data Commons. One such feature is the new Data Download tool that allows you to easily download statistical variable data for a large number of places with just a few button clicks.
```

Comment:
nit: we should link to the data download tool :)
```suggestion
In the last year, we have added several interesting datasets and exciting new features to Data Commons. One such feature is the new [Data Download tool](...) that allows you to easily download statistical variable data for a large number of places with just a few button clicks.
```

Reply:
done